composer.json

```
@@ -5,10 +5,10 @@
     "description": "Symfony survival kit",
     "require": {
         "php": "^7.4",
-        "symfony/flex": "^1.0",
-        "symfony/framework-bundle": "^4.1",
-        "sensio/framework-extra-bundle": "^5.2",
-        "symfony/monolog-bundle": "^3.6",
+        "symfony/flex": "^1.3.1",
+        "symfony/framework-bundle": "5.3.*",
+        "sensio/framework-extra-bundle": "6.1.*",
+        "symfony/monolog-bundle": "3.*",
```

it is better to allow all backward compatible versions
```suggestion
        "symfony/flex": "^1.3",
        "symfony/framework-bundle": "^5.3",
        "sensio/framework-extra-bundle": "^6.1",
        "symfony/monolog-bundle": "^3.0",
```